pkg/server/registry/apigroups/acorn/events/strategy.go

```diff
@@ -158,13 +194,18 @@ func (q query) filter(events ...apiv1.Event) []apiv1.Event {
 func stripQuery(opts storage.ListOptions) (q query, stripped storage.ListOptions, err error) {
 	stripped = opts
 
+	now := internalv1.MicroTime(metav1.NowMicro())
```

Comment:
nit: can we use the `NowMicro` function that was defined also in this PR?

Reply:
done